themes-default/slim/static/js/templates/name-pattern.vue

```diff
@@ -398,63 +424,89 @@ module.exports = {
             const { $el } = this;
             const el = $($el);
 
-            const result = apiRoute.get('config/postProcessing/isNamingValid', { params }).then(res => {
-                if (result.data === 'invalid') {
-                    el.find('#naming_pattern').qtip('option', {
-                        'content.text': 'This pattern is invalid.',
-                        'style.classes': 'qtip-rounded qtip-shadow qtip-red'
-                    });
-                    el.find('#naming_pattern').qtip('toggle', true);
-                    el.find('#naming_pattern').css('background-color', '#FFDDDD');
-                } else if (result.data === 'seasonfolders') {
-                    el.find('#naming_pattern').qtip('option', {
-                        'content.text': 'This pattern would be invalid without the folders, using it will force "Flatten" off for all shows.',
-                        'style.classes': 'qtip-rounded qtip-shadow qtip-red'
-                    });
-                    el.find('#naming_pattern').qtip('toggle', true);
-                    el.find('#naming_pattern').css('background-color', '#FFFFDD');
-                } else {
-                    el.find('#naming_pattern').qtip('option', {
-                        'content.text': 'This pattern is valid.',
-                        'style.classes': 'qtip-rounded qtip-shadow qtip-green'
-                    });
-                    el.find('#naming_pattern').qtip('toggle', false);
-                    el.find('#naming_pattern').css('background-color', '#FFFFFF');
-                }
-            }).catch(error => {
-                console.warn(error);
-            });
+            apiRoute.get('config/postProcessing/isNamingValid', { params })
+                .then(result => {
```

Comment:
I had that before the linter started complaining

Reply:
What about? We don't have any line length or anything.

Reply:
![image](https://user-images.githubusercontent.com/1331394/43364278-6df16092-9317-11e8-853b-3b6227dd6591.png)

Reply:
What code are you using? I just did the following and got no linting errors:
```js
            apiRoute.get('config/postProcessing/isNamingValid', { params }).then(result => {
                if (result.data === 'invalid') {
                    el.find('#naming_pattern').qtip('option', {
                        'content.text': 'This pattern is invalid.',
                        'style.classes': 'qtip-rounded qtip-shadow qtip-red'
                    });
                    el.find('#naming_pattern').qtip('toggle', true);
                    el.find('#naming_pattern').css('background-color', '#FFDDDD');
                } else if (result.data === 'seasonfolders') {
                    el.find('#naming_pattern').qtip('option', {
                        'content.text': 'This pattern would be invalid without the folders, using it will force "Flatten" off for all shows.',
                        'style.classes': 'qtip-rounded qtip-shadow qtip-red'
                    });
                    el.find('#naming_pattern').qtip('toggle', true);
                    el.find('#naming_pattern').css('background-color', '#FFFFDD');
                } else {
                    el.find('#naming_pattern').qtip('option', {
                        'content.text': 'This pattern is valid.',
                        'style.classes': 'qtip-rounded qtip-shadow qtip-green'
                    });
                    el.find('#naming_pattern').qtip('toggle', false);
                    el.find('#naming_pattern').css('background-color', '#FFFFFF');
                }
            }).catch(error => {
                console.warn(error);
            });
```